core/workspace_svg.js

```diff
@@ -961,6 +981,18 @@ Blockly.WorkspaceSvg.prototype.paste = function(xmlBlock) {
   if (this.currentGesture_) {
     this.currentGesture_.cancel();  // Dragging while pasting?  No.
   }
+  if (xmlBlock.tagName.toLowerCase() == 'comment') {
```

Comment:
Paste can now paste blocks and ws comments.

Reply:
Hmmm does this need to be enabled in the vertical_playground? I had trouble getting comment copy-paste working.

Reply:
https://github.com/google/blockly/issues/1867